test/functional/wallet_basic.py

```diff
@@ -255,7 +255,7 @@ def run_test(self):
         assert(txid1 in self.nodes[3].getrawmempool())
 
         # Exercise balance rpcs
-        assert_equal(self.nodes[0].getwalletinfo()["unconfirmed_balance"], 1)
+        assert_equal(self.nodes[0].getwalletinfo()["unconfirmed_balance"]['00'], 1)
```

Comment:
There are no test case for colored balance?

Reply:
Yes, there are no functionality tests for colored coin balance.
I think it should be added with a task that is adding colored balance support for each RPCs. see https://github.com/Naviabheeman/tapyrus-core/wiki/Colored-coin-support-in-Tapyrus-Wallet#task-3-balance-reporting-rpcs